deeppavlov/models/morpho_tagger/morpho_tagger.py

```diff
@@ -13,30 +13,39 @@
 # limitations under the License.
 
 from logging import getLogger
-from typing import List, Union, Tuple, Iterable
+from pathlib import Path
+from typing import List, Optional, Union, Tuple
 
+import numpy as np
 import keras.layers as kl
 import keras.optimizers as ko
 import keras.regularizers as kreg
+import keras.backend as kb
 from keras import Model
+from keras import backend as K
```

Comment:
It's weird to import `keras.backend` twice. You can replace all keras imports with a line
```python3
from keras import Model, backend as kb, layers as kl, regularizers as kreg, optimizers as ko
```
and replace one usage of `K` with `kb`

Reply:
indeed, fixed